.gitignore

```
@@ -7,4 +7,5 @@ node_modules
 e2e-testing/.resources
 e2e-testing/.test-extensions
 test-reports/
-tsconfig.tsbuildinfo
\ No newline at end of file
+tsconfig.tsbuildinfo
+.env.local
```

May as well .gitignore all .local files, as we may want other dotenv environments in the future.
```suggestion
.env*.local
```